samples/matter/weather_station/README.rst

```diff
@@ -80,6 +80,17 @@ If you want to build the target with the debug configuration, run the following
 
    west build -b thingy53_nrf5340_cpuapp -- -DCMAKE_BUILD_TYPE=debug
 
+Device Firmware Upgrade support
+===============================
+
+This sample by default uses the secure bootloader for performing over-the-air Device Firmware Upgrade using Bluetooth LE.
+To build the sample with DFU support disabled, run the following command:
```

Comment:
```suggestion
To build the sample with both the secure bootloader and the DFU support disabled, run the following command:
```

Reply:
You can also follow the changes from https://github.com/nrfconnect/sdk-nrf/pull/5157.